apps/nestjs-api/src/reminder-emails/reminder-emails.service.ts

```diff
@@ -366,100 +499,152 @@ export class ReminderEmailsService {
-  //       ? // Mentor Emails for 2 and 4 weeks
-  //         mentorshipMatchAgeInDays === 14
-  //         ? 'Mentor_Log_Mentoring_Sessions_Reminder_1_1711110740940'
-  //         : 'Mentor_Log_Mentoring_Sessions_Reminder_2_1711112496532'
-  //       : // Mentee Emails for 2 and 4 weeks
-  //       mentorshipMatchAgeInDays === 14
-  //       ? 'Mentee_Log_Mentoring_Sessions_Reminder_1_1711114670729'
-  //       : 'Mentee_Log_Mentoring_Sessions_Reminder_2_1711115347143'
-
-  //   const template = await this.emailTemplatesService.getEmailTemplate(
-  //     sfEmailTemplateDeveloperName
-  //   )
-
-  //   if (!template) {
-  //     throw new Error(
-  //       `Email template not found: ${sfEmailTemplateDeveloperName}`
-  //     )
-  //   }
+  async sendLogMentoringSessionsReminder({
+    email,
+    firstName,
+    userType,
+    mentorshipMatchAgeInDays,
+  }) {
+    const sfEmailTemplateDeveloperName =
+      userType === UserType.MENTOR
+        ? // Mentor Emails for 2 and 4 weeks
+          mentorshipMatchAgeInDays === 14
+          ? 'Mentor_Log_Mentoring_Sessions_Reminder_1_1711110740940'
+          : 'Mentor_Log_Mentoring_Sessions_Reminder_2_1711112496532'
+        : // Mentee Emails for 2 and 4 weeks
+        mentorshipMatchAgeInDays === 14
+        ? 'Mentee_Log_Mentoring_Sessions_Reminder_1_1711114670729'
+        : 'Mentee_Log_Mentoring_Sessions_Reminder_2_1711115347143'
 
-  //   const sanitizedHtml = template.HtmlValue.replace(
-  //     /{{{Recipient\.FirstName}}}/g,
-  //     `${firstName} ${email}`
-  //   )
+    const template = await this.emailTemplatesService.getEmailTemplate(
+      sfEmailTemplateDeveloperName
+    )
 
-  //   const params = {
-  //     Destination: {
-  //       ToAddresses: isProductionOrDemonstration()
-  //         ? ['anilakarsu93@gmail.com']
-  //         : [this.config.get('NX_DEV_MODE_EMAIL_RECIPIENT')],
-  //     },
-  //     Message: {
-  //       Body: {
-  //         Html: { Data: sanitizedHtml },
-  //       },
-  //       Subject: { Data: template.Subject },
-  //     },
-  //     Source: 'career@redi-school.org',
-  //   }
+    if (!template) {
+      throw new Error(
+        `Email template not found: ${sfEmailTemplateDeveloperName}`
+      )
+    }
 
-  //   this.ses.sendEmail(params, (err, data) => {
-  //     if (err) console.log(err, err.stack)
-  //     else console.log(data)
-  //   })
+    const sanitizedHtml = template.HtmlValue.replace(
+      /{{{Recipient\.FirstName}}}/g,
+      `${firstName} ${email}`
+    )
 
-  //   return { message: 'Email sent' }
-  // }
+    const params = {
+      Destination: {
+        ToAddresses: isProductionOrDemonstration()
+          ? ['anilakarsu93@gmail.com']
+          : [this.config.get('NX_DEV_MODE_EMAIL_RECIPIENT')],
+      },
+      Message: {
+        Body: {
+          Html: { Data: sanitizedHtml },
+        },
+        Subject: { Data: template.Subject },
+      },
+      Source: 'career@redi-school.org',
```

Comment:
We can pass the `SENDER_EMAIL` variable also here.

Reply:
Fixed!